src/views/primary-auth/CustomButtons.js

```diff
@@ -27,16 +27,15 @@ define([
 
   return Okta.View.extend({
 
-    className: 'social-auth',
+    className: 'primary-auth-container',
 
     children: function () {
       var children = [],
-          idProviders = this.settings.get('configuredSocialIdps'),
-          divider = dividerTpl({text: Okta.loc('socialauth.divider.text', 'login')});
+        buttons = this.settings.get('configuredButtons'),
+        divider = dividerTpl({text: Okta.loc('authbutton.divider.text', 'login')});
```

Comment:
Spoke to @mauriciocastillosilva-okta about this. I'll change it back to use socialauth.divider.text.